packages/fxa-customs-server/.eslintrc

```diff
@@ -4,10 +4,12 @@
   "parserOptions": {
     "ecmaFeatures": {
       "globalReturn": true
-    }
+    },
+    "sourceType": "module"
   },
   "rules": {
     "strict": "off",
-    "handle-callback-err": "off"
+    "handle-callback-err": "off",
+    "camelcase": "off"
```

Comment:
Q: Is this just because of the two new files in docs/swagger/*?

Reply:
The [.eslintrc file in fxa-content-server](https://github.com/mozilla/fxa/blob/main/packages/fxa-content-server/.eslintrc), which also includes `extends: ["plugin:fxa-client"]`, eliminated the camelcase problem in its own pm2.config.js file by adding `"camelcase": "off"` as well.